src/core/services/aria/aria.js

```diff
@@ -62,7 +62,26 @@ function AriaService($$rAF, $log, $window, $interpolate) {
   }
 
   function getText(element) {
-    return (element.text() || "").trim();
+    element = element[0] || element;
+    var walker = document.createTreeWalker(element, NodeFilter.SHOW_TEXT, null, false);
+    var text = '';
+
+    var node;
+    while (node = walker.nextNode()) {
```

Comment:
@topherfangio I intentionally didn't use the filter for the `TreeWalker`, because it's not supported by IE browsers and this will break ngMaterial on IE browsers. 

Additionally, the filter is actually doing the same as me, just a shorter code + The amount of nodes is really minimized, because we're only iterating through the text nodes.